cmd/ctr/commands/run/run.go

```diff
@@ -119,7 +119,7 @@ func replaceOrAppendEnvValues(defaults, overrides []string) []string {
 var Command = cli.Command{
 	Name:      "run",
 	Usage:     "run a container",
-	ArgsUsage: "[flags] Image|RootFS ID [COMMAND] [ARG...]",
```

Comment:
nit: do you mind changing `ID` to `CONTAINER`
nit: do you mind updating the error message in L185 to just `"ref must be provided"`